### ISSUE TITLE
1.9.12 - Adding Collaborators on Internal Note

### ISSUE THREAD

**Issue by davideakc** [osTicket/osTicket-1.8#2716]
Hi,

I want to ask if it's possibile with a code, add the collaborators (like on the "Post Reply" tab) even on the "Internal Note" because I wanna notify more than one person that there is a new internal activity.

(i don't want do it with the "Teams" because in my system there are more than 100 agents and it's impossible to make teams for every agent)...

I hope in your help, thanks.

**Comment by ssellmeier** [comment]
Hello,

I looked at the files relating to the "post Note" feature and can't find a "code" or something for this. So as far as I can see - this will become a feature request?

**Comment by davideakc** [comment]
So for now there's no code or something for doing it?

**Comment by ssellmeier** [comment]
No, I don't see any implementation of such a feature in the code!

I'm completely new to the osTicket project as community-member as well as possible contributor. So I don't know the rules and work-flows here! There are two ways for me to implement such a feature:
 * Implementing a "Tag-System" which parses the notes-text for "@staff-username" in this syntax or in another
 * Implementing a new field to optional notify specified staff-members

**Comment by davideakc** [comment]
Yes, "Implementing a new field to optional notify specified staff-members" would be very useful!

**Comment by markus4000** [comment]
Or a "star" or "follow" button for staff. So an agent can "follow" a ticket if he wants.

**Comment by davideakc** [comment]
So for now there's no way to do something? (experimental)
